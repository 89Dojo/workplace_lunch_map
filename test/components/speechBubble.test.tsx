import { render } from 'test/testUtils'
import WithSpeechBubbles from 'src/components/speechBubble'
import * as React from 'react'

describe('Speech Bubble', () => {
  it('Snapshot Test', () => {
    const { container } = render(<WithSpeechBubbles />)
    expect(container).toMatchSnapshot()
  })
})
