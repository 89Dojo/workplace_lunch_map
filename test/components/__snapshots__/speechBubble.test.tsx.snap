// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Speech Bubble Snapshot Test 1`] = `
<div>
  <div
    class="css-7luvuw"
  >
    <div
      class="chakra-stack chakra-container css-1cvohfv"
    >
      <div
        class="chakra-stack css-jp20jv"
      >
        <h2
          class="chakra-heading css-1dklj6k"
        >
          Our Clients Speak
        </h2>
        <p
          class="chakra-text css-0"
        >
          It's a big lie, but here's what our customers have to say.
        </p>
      </div>
      <div
        class="chakra-stack css-1k0eqqv"
      >
        <div
          class="css-0"
        >
          <div
            class="chakra-stack css-iqyly9"
          >
            <h3
              class="chakra-heading css-pvm5ii"
            >
              Ready to use
            </h3>
            <p
              class="chakra-text css-3c0zu9"
            >
              I can't wait to use it as soon as I think of it.
            </p>
          </div>
          <div
            class="css-1p6ys5y"
          >
            <span
              alt="Jane Cooper"
              class="chakra-avatar css-1j8889u"
            >
              <svg
                aria-label=" avatar"
                class="chakra-avatar__svg css-16ite8i"
                role="img"
                viewBox="0 0 128 128"
              >
                <path
                  d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                  fill="currentColor"
                />
                <path
                  d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                  fill="currentColor"
                />
              </svg>
            </span>
            <div
              class="chakra-stack css-130k5dj"
            >
              <p
                class="chakra-text css-35ezg3"
              >
                Jane Cooper
              </p>
              <p
                class="chakra-text css-1pahln7"
              >
                CEO at ABC Corporation
              </p>
            </div>
          </div>
        </div>
        <div
          class="css-0"
        >
          <div
            class="chakra-stack css-iqyly9"
          >
            <h3
              class="chakra-heading css-pvm5ii"
            >
              Easy to use
            </h3>
            <p
              class="chakra-text css-3c0zu9"
            >
              I love how easy it is to create a web page.
            </p>
          </div>
          <div
            class="css-1p6ys5y"
          >
            <span
              alt="Jane Cooper"
              class="chakra-avatar css-1j8889u"
            >
              <svg
                aria-label=" avatar"
                class="chakra-avatar__svg css-16ite8i"
                role="img"
                viewBox="0 0 128 128"
              >
                <path
                  d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                  fill="currentColor"
                />
                <path
                  d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                  fill="currentColor"
                />
              </svg>
            </span>
            <div
              class="chakra-stack css-130k5dj"
            >
              <p
                class="chakra-text css-35ezg3"
              >
                Jane Cooper
              </p>
              <p
                class="chakra-text css-1pahln7"
              >
                CEO at ABC Corporation
              </p>
            </div>
          </div>
        </div>
        <div
          class="css-0"
        >
          <div
            class="chakra-stack css-iqyly9"
          >
            <h3
              class="chakra-heading css-pvm5ii"
            >
              Free to use
            </h3>
            <p
              class="chakra-text css-3c0zu9"
            >
              It's free, so it's very nice and safe for poor people to use.
            </p>
          </div>
          <div
            class="css-1p6ys5y"
          >
            <span
              alt="Jane Cooper"
              class="chakra-avatar css-1j8889u"
            >
              <svg
                aria-label=" avatar"
                class="chakra-avatar__svg css-16ite8i"
                role="img"
                viewBox="0 0 128 128"
              >
                <path
                  d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                  fill="currentColor"
                />
                <path
                  d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                  fill="currentColor"
                />
              </svg>
            </span>
            <div
              class="chakra-stack css-130k5dj"
            >
              <p
                class="chakra-text css-35ezg3"
              >
                Jane Cooper
              </p>
              <p
                class="chakra-text css-1pahln7"
              >
                CEO at ABC Corporation
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
